src/components/CreateClusterTemplateWizard/CreateClusterTemplateWizard.tsx

```diff
@@ -82,9 +126,11 @@ const _CreateClusterTemplateWizard = () => {
 
 const CreateClusterTemplateWizard = () => (
   <NamespacesContextProvider>
-    <AccessContextProvider>
-      <_CreateClusterTemplateWizard />
-    </AccessContextProvider>
+    <UsersContextProvider>
+      <QuotasContextProvider>
```

Comment:
all these context are slowing down the wizard rendering. Why do we need to wait for quotas, users and namespaces to render the first page which does not need any of that info ?

Reply:
Yes you are right.
I'm thinking of the best way to resolve this.

For example, we need QuotasContextProvider in the following places:
wizard review
wizard quotas step
quotas tab in cluster templates list page
quotas sub tab of a cluster templates details page

One option is to wrap each component like this with the context, but then the data won't be saved for the other components

I suggest to put the contexts providers in ClusterTemplatesPage
once user enters the page the data will be requested but the page will render regardless 
the contexts will have a method load and error, and the components that need them will show loading/error when needed

WDYT?